src/source/Signaling/LwsApiCalls.c

```diff
@@ -832,6 +924,12 @@ STATUS createChannelLws(PSignalingClient pSignalingClient, UINT64 time)
                                  SIGNALING_SERVICE_API_CALL_CONNECTION_TIMEOUT, SIGNALING_SERVICE_API_CALL_COMPLETION_TIMEOUT,
                                  DEFAULT_LOW_SPEED_LIMIT, DEFAULT_LOW_SPEED_TIME_LIMIT, pSignalingClient->pAwsCredentials, &pRequestInfo));
 
+    if (pSignalingClient->signalingClientCallbacks.getCurrentTimeFn != NULL) {
+        pRequestInfo->currentTime = pSignalingClient->signalingClientCallbacks.getCurrentTimeFn(pSignalingClient->signalingClientCallbacks.customData);
+    }
+
+    checkAndCorrectForClockSkew(pSignalingClient, pRequestInfo);
```

Comment:
Shouldnt we be checking for `STATUS` here? What happens in the event `hashTableGet` fails?

Reply:
I intentionally left it out, only one of two things happen, either its in the hash table (which means we apply the offset, there's no status failures possible in that route), other case is its not in the hash table in which case it CHK_STATUS will return some non success status but all that means is that there was no clock skew so nothing to correct.